lib/grainstore/index.js

```diff
@@ -5,5 +5,5 @@ module.exports = {
         return version;
     },
     MMLStore: require('./mml_store'),
-    MMLBuilder: require('./mml_builder')
+    MMLBuilder: require('./mml-builder/mml-builder-worker')
```

Comment:
Quick question: should we keep the old interface available? I mean, with this change we only keep the `toXML` method in the public interface. Most likely, we don't need to expose `baseMML` alike methods, however, I'm not that sure about `set` method.